Gemfile

```diff
@@ -15,9 +15,10 @@ gem 'virtus'
 gem 'rails-api'
 gem 'figaro'
 gem 'pg'
-
 gem 'active_model_serializers', '~> 0.10.0'
-
+gem 'will_paginate'
+gem 'responders'
```

Comment:
I'm not that familiar with this gem, but it doesn't look like we are really using it as we just respond with JSON to every request